src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+    background: url("../images/star.svg");
+    @include backgroundImage;
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+  @for $star from 0 through 5 {
+    &--#{$star} :nth-child(-n + #{$star}) {
+      background: url("../images/star-active.svg");
+      background-repeat: no-repeat;
+      background-position: center;
```

Comment:
```suggestion
      background-image: url("../images/star-active.svg");
```
There is no need to duplicate all properties

Reply:
Hi, I did it, because when I don`t use this properties, my stars start repeating.  I hope it`s okay that I`ll use mixin instead of this duplicating bg-properties)
![image](https://github.com/mate-academy/layout_catalog/assets/63805271/c41254bb-ab8e-4b0a-b7f4-0d31b1dfc0ce)